src/typechecker.erl

```diff
@@ -4846,7 +4846,7 @@ type_check_form(Function, Errors, StopOnFirstError, Env, Opts)
     catch
         throw:Throw:St ->
             % Useful for debugging
-            % io:format("~p~n", [erlang:get_stacktrace()]),
+            % io:format("~p~n", [St]),
             if
                 CrashOnError ->
                     {crash, Throw, St};
```

Comment:
I think these commands can be deleted. We have CrashOnError which returns the stacktrace instead. I assume these comments are from before that.

Btw, the if expression could be incorporated in the catch clauses as guards, e.g.

```Erlang
        throw:Throw:St when CrashOnError ->
            {crash, Throw, St};
        throw:Throw ->
            {errors, [Throw | Errors]};
        error:Error:ST ->
            ...
```